k8s/namespaces/fees-pay/ccpay-refunds-api/prod.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ccpay-refunds-api
+spec:
+  values:
+    java:
+      image: hmctspublic.azurecr.io/ccpay/refunds-api:prod #{"$imagepolicy": "flux-system:demo-ccpay-refunds-api"}
```

Comment:
Hi, can you review this - prod has been set to use your demo image policy `demo-ccpay-refunds-api` which is a pr- image